charts/diskmanager/templates/secrets.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: secrets-manager.tuenti.io/v1alpha1
+kind: SecretDefinition
+metadata:
+  name: {{ .Values.name }}-gcp-sa-secretdefinition
+  labels:
+    {{ include "diskmanager.labels" . | nindent 4 }}
+spec:
+  name: {{ .Values.name }}-gcp-sa
+  keysMap:
+    application_default_credentials.json:
+      path: {{ .Values.vault.pathPrefix }}/diskmanager/terraform/diskmanager-sa
```

Comment:
I'd add a required condition to the pathPrefix, since this resource will be invalid if it is omitted